### PR TITLE
Team1 revision of guides

### PR DESCRIPTION
**Pull request by JMilamber**
This Pull Request adds the Revision of Guides: Assessment and Code of Conduct Revision Policies to assessment.md. 
Both Travis CI checks now pass.
Let me know if there is anything to change.

### CODE REVIEW

File: assessment.md [hunk 1]
```diff
@@ -100,4 +100,30 @@ Project Management is Covered by Team 2
 ## Revision of Guides
 
 * Policies for revising the assessment form
+  * 😞 = Revision on unassigned and assigned sections of the assessment form
+      are made without approval, a commit message, and a Pull Request
+  * 😕 = Revision on assigned sections of the assessment form are made with
+      approval. Revisions lack an adequate commit message and a Pull Request
+  * 😐 = Revision on assigned sections of the assessment form are made with
+      approval, an adequate commit message, and a Pull request that tags a TD or
```
Comment (enpuyou) [inline assessment.md]: The use of "Pull Request" and "Pull request" is inconsistent in both sections

Comment (enpuyou) [inline assessment.md]: I think you mean technical leader here. There is also no hyphen in "project manager"

**Review by enpuyou — CHANGES_REQUESTED** [review]
A few fixes need to be made. Also, in general, is the use of "a commit message" necessary here? Since there are other sections that handle the assessment of commit messages. Instead of specifying the commit message, maybe just clarify that the GitHub flow model and other good practices should be conformed to when revising an assessment document. This is just my opinion, other TLs can also comment on this!

File: assessment.md [hunk 2]
```diff
@@ -136,7 +136,39 @@
...
   leaders to seek advice and help when necessary
 
-## Revision of Guides
+### Revision of Guides
+
+This section describes the policies for revising the assessment form and Code of
+Conduct. Revision on either of these two repositories must be approved by
+project leader and follow the GitHub Flow Model. Additionally, Pull requests for
+revision of either guide must tag multiple TL's, a project leader, and another
+classmate.
```
Comment (enpuyou) [inline assessment.md]: This section should be removed according to the latest assessment template

**Review by enpuyou — CHANGES_REQUESTED** [review]
The content looks great, just two more fixes need to be made. In addition to the one below, the use of emoji is not correct according to the assessment template

File: assessment.md [hunk 3]
```diff
@@ -193,7 +193,36 @@
   * :sunglasses: = Polite, patient, and effectively communicates with technical
   leaders to seek advice and help when necessary
 
-## Revision of Guides
+### Revision of Guides
```
Comment (enpuyou) [inline assessment.md]: I believe that the header should be ## instead of ###

**Review by enpuyou — CHANGES_REQUESTED** [review]
Looks good. One more small fix need to make

File: assessment.md (hunk 3 above)
Comment (JMilamber) [reply assessment.md]: In the Markdown document Subcategories (like revision of guides) are denoted with three hashtags I think , and its the Overall category (I.e. Technical or Professional) that have two hashtags

Comment (enpuyou) [reply assessment.md]: What you are saying is correct, however, `Revision of Guides` here is not a `subcategory` as I have found in this <https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-assessment/commit/d73b1f9f70c527e9ca647000e017a3449543b05a> of the master branch. Thus, it should be in the same level as `Technical` or `Professional`

Comment (JMilamber) [reply assessment.md]: Ohhh okay that makes sense, thanks Enpu!

Comment (enpuyou) [reply assessment.md]: No problem!

**Review by enpuyou — APPROVED** [review]

**Review by ibarrezueta — APPROVED** [review]

**Review by cklima616 — COMMENTED** [review]
Formatting looks consistent, and mdl checks have passed.

**Review by cklima616 — APPROVED** [review]

**Review by bagashvilit — APPROVED** [review]

**Pull request merged**